### PR TITLE
Fix RNA pipeline 

### LINKED ISSUE #48: Use RNA/DNA reference genomes correctly.

### PR DESCRIPTION
**Pull request by apeltzer**

### CODE REVIEW

**Comment by apeltzer** [comment]
This should fix the #48 issue 👍

File: main.nf [hunk 1]
```diff
@@ -207,10 +207,12 @@ if ( !params.bam  ) { // FASTQ files processing
         set val(pattern), "mapped_{1,2}.bam" into fished_reads
 
         script:
+        index = (params.seqtype =='dna') ?  "$baseDir/data/indices/yara/hla_reference_dna" :  "$baseDir/data/indices/yara/hla_reference_rna"
```
Comment (MaxUlysse) [inline main.nf]: index = $baseDir/data/indices/yara/hla_reference_${params.seqtype} ?

Comment (MaxUlysse) [reply main.nf]: I'm guessing you have tests somewhere to ensure the seqtype is only `rna` or `dna` ;-)

**Comment by apeltzer** [comment]
Unfortunately the r dependencies are not present in the container atm, so we'd need to build locally first.

**Pull request merged**